django/template/loader_tags.py

```diff
@@ -207,6 +210,11 @@ def render(self, context):
         except Exception:
             if context.template.engine.debug:
                 raise
+            else:
+                template_name = getattr(context, 'template_name', 'unknown')
+                logger.warn('Exception raised while rendering {{% include %}} for '
```

Comment:
yeah I'll get them up today.
On Sep 21, 2015 5:20 AM, "Tim Graham" <notifications@github.com> wrote:

> In django/template/loader_tags.py
> <https://github.com/django/django/pull/5285#discussion_r39964078>:
>
> > @@ -207,6 +210,11 @@ def render(self, context):
> >          except Exception:
> >              if context.template.engine.debug:
> >                  raise
> > +            else:
> > +                template_name = getattr(context, 'template_name', 'unknown')
> > +                logger.warn('Exception raised while rendering {{% include %}} for '
>
> Do you have time to complete these changes today? Otherwise, I'll try to
> take care of it before the feature freeze for 1.9.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/django/django/pull/5285/files#r39964078>.
>